foreman/data_refinery_foreman/surveyor/sra.py

```diff
@@ -300,6 +303,21 @@ def _build_file_url(run_accession: str, read_suffix=""):
             long_accession=run_accession,
             read_suffix=read_suffix)
 
+    @staticmethod
+    def _build_ncbi_file_url(run_accession: str):
+        # ENA has a weird way of nesting data: if the run accession is
```

Comment:
Is this still true outside of `ENA`? If so, this should probably say NCBI SRA. If not, this comment may be removable.